thumbnail-deploy-xopera/docker-image-templates/image-resize-template/function/handler.py

```diff
@@ -0,0 +1,41 @@
+import json
+from PIL import Image
+from .minio_handler import MinIO
+import os
+
+THUMBNAIL_SIZES_PX = [100, 160, 200]
+
+node_ip = "10.10.43.213"
```

Comment:
Again IP address.

Reply:
I replaced this with documentational IP.